src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GetTimelineTest.kt

```diff
@@ -136,11 +136,11 @@ class GetTimelineTest : IntegrationTestBase() {
         .returnResult(TimelineResponse::class.java)
 
       // Then
-      val actual = response.responseBody.blockFirst()
-      assertThat(actual.events).hasSize(4)
+      val actual = response.responseBody.blockFirst()!!
       val actionPlanCreatedCorrelationId = actual.events[3].correlationId
       assertThat(actual)
         .isForPrisonNumber(prisonNumber)
+        .hasNumberOfEvents(4)
```

Comment:
We already had a method on the assertion class to assert the number of events, so used this rather than `assertThat(actual.events).hasSize(4)` (which was moaning about potential non-null safety)